avalon/maya/lib.py

```diff
@@ -110,6 +115,12 @@ def export_alembic(nodes,
 
     """
 
+    if frame_range is None:
+        frame_range = (
+            cmds.playbackOptions(query=True, ast=True),
+            cmds.playbackOptions(query=True, aet=True)
```

Comment:
I think we should use full keyword arguments unless the abbreviation is clear enough such as `add` instead of `addNamespace`. It's a minor fix but worth mentioning.

Reply:
Totally agree. In this case I solely moved the lines of code up a bit, they are not new. I tried to change as little as possible because we are not the only ones managing the avalon core code and this was written by someone else, I assume marcus. :)